### PR TITLE
Adding Desktop to System.IO.Packaging

### PR DESCRIPTION
**Pull request by venkat-raman251**

### CODE REVIEW

**Comment by venkat-raman251** [comment]
@weshaggard : Please have a look.

**Comment by weshaggard** [comment]
@dotnet-bot Innerloop OpenSUSE13.2 Release Build and Test

File: src/System.IO.Packaging/src/System.IO.Packaging.csproj [hunk 1]
```diff
@@ -9,17 +9,18 @@
     <AssemblyVersion>4.0.0.0</AssemblyVersion>
     <RootNamespace>System.IO.Packaging</RootNamespace>
     <AssemblyName>System.IO.Packaging</AssemblyName>
-    <PackageTargetFramework>dotnet5.4</PackageTargetFramework>
```
Comment (weshaggard) [inline src/System.IO.Packaging/src/System.IO.Packaging.csproj]: Why did you remove this? I would expect it should stay but you should add the condition `'$(PackageTargetFramework)' == ''`

File: src/System.IO.Packaging/src/project.json [hunk 2]
```diff
@@ -1,14 +1,20 @@
...
+        "System.IO.Compression": "4.0.0",
+        "System.IO.FileSystem": "4.0.0",
+        "System.IO.FileSystem.Primitives": "4.0.0",
+        "System.Runtime": "4.0.20",
+        "System.Xml.ReaderWriter": "4.0.0"
+      }
+    },
+    "net46":{
+      "dependencies": {
+        "Microsoft.TargetingPack.NetFramework.v4.6": "1.0.0-rc2-23530",
```
Comment (weshaggard) [inline src/System.IO.Packaging/src/project.json]: Remove the "-rc2-253530" prerelease on this.

File: src/System.IO.Packaging/src/System.IO.Packaging.csproj [hunk 3]
```diff
@@ -47,5 +48,9 @@
     <Compile Include="System\IO\Packaging\ZipStreamManager.cs" />
     <Compile Include="System\IO\Packaging\ZipWrappingStream.cs" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='net46'">
```
Comment (weshaggard) [inline src/System.IO.Packaging/src/System.IO.Packaging.csproj]: Shouldn't this be a partial facade on net46?

Comment (venkat-raman251) [inline src/System.IO.Packaging/src/System.IO.Packaging.csproj]: @weshaggard : Commited the changes. Please check.

File: src/System.IO.Packaging/src/project.json [hunk 4]
```diff
@@ -1,14 +1,20 @@
...
-    "System.Diagnostics.Debug": "4.0.10",
-    "System.IO.Compression": "4.0.0",
-    "System.IO.FileSystem": "4.0.0",
-    "System.IO.FileSystem.Primitives": "4.0.0",
-    "System.Runtime": "4.0.20",
-    "System.Xml.ReaderWriter": "4.0.0"
-  },
   "frameworks": {
-    "dnxcore50": {}
+    "dnxcore50": {
```
Comment (weshaggard) [inline src/System.IO.Packaging/src/project.json]: New convention. Can you please change dnxcore50 to dotnet5.4 for this project to match the default PackageTargetFramework

**Comment by weshaggard** [comment]
LGTM beyond the one tweak in the project.json.

**Comment by venkat-raman251** [comment]
@weshaggard : Updated project.json to dotnet5.4

**Comment by weshaggard** [comment]
LGTM merge after green CI.

**Comment by weshaggard** [comment]
@dotnet-bot test this

**Comment by weshaggard** [comment]
@dotnet-bot test this please

**Comment by ericstj** [comment]
@dotnet-bot test  Innerloop CentOS7.1 Release Build and Test

**Comment by ericstj** [comment]
You can ignore the OpenSUSE13.2 failure.  We've disable the OpenSUSE13.2 build in newer PRs.  Feel free to merge this.
